internal/third_party/dep/fs/fs.go

```diff
@@ -69,20 +70,24 @@ func renameByCopy(src, dst string) error {
 	if dir, _ := IsDir(src); dir {
 		cerr = CopyDir(src, dst)
 		if cerr != nil {
-			cerr = errors.Wrap(cerr, "copying directory failed")
+			cerr = fmt.Errorf("copying directory failed: %w", cerr)
 		}
 	} else {
 		cerr = copyFile(src, dst)
 		if cerr != nil {
-			cerr = errors.Wrap(cerr, "copying file failed")
+			cerr = fmt.Errorf("copying file failed: %w", cerr)
 		}
 	}
 
 	if cerr != nil {
-		return errors.Wrapf(cerr, "rename fallback failed: cannot rename %s to %s", src, dst)
+		return fmt.Errorf("rename fallback failed: cannot rename %s to %s: %w", src, dst, cerr)
+	}
+
+	if cerr = os.RemoveAll(src); cerr != nil {
+		return fmt.Errorf("cannot delete %s: %w", src, cerr)
```

Comment:
```suggestion
	if err := os.RemoveAll(src); err != nil {
		return fmt.Errorf("cannot delete %s: %w", src, err)
```

nit: no need to continue to "reuse" cerr here (very minor)